Src/AutoFixture/EmailAddressLocalPart.cs

```diff
@@ -0,0 +1,142 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace Ploeh.AutoFixture
+{
+    /// <summary>
+    /// Represents the local part of the email address, defined as everything up to, but not including the @ sign.  
+    /// The requirements enforced on the local part are defined in RFC 3696.  The regular expression for validating 
+    /// the local part borrowed from Phil Haack at 
+    /// http://haacked.com/archive/2007/08/21/i-knew-how-to-validate-an-email-address-until-i.aspx/
```

Comment:
So in terms of the local part only and the `EmailAddressLocalPart` constructor, I expect the first 4 to fail.  The local part cannot contain an @ sign and those do. While otherwise presumably legal email addresses, they don't conform to the requirement for the local part specifically and that's what `EmailAddressLocalPart` validates for.   When I tested it, "f#ers.com" is accepted as a valid local part as # is allowed and the .com has no particular significance to the local part.  

It occurs to me that perhaps it would be useful to have a way to create `EmailAddressLocalPart` from a complete email address as right now it would have to be done by hand. 

All the above having been said, you're right that the regular expression overall is not liberal enough in that it fails for the characters you included that are outside of the [A-Za-z0-9] set.  However, I'm unsure if the RFC actually allows for the addresses you posted and if it doesn't, what we should do about it.   I came across http://isemail.info/about which seems to have spent an intense amount of time studying the issue and when I pasted in the first 4 addresses you provided, that site rules all of them as invalid based on the non-US ASCII characters.  

Assuming the rules as enforced by that site are truly exact to the RFC, the larger question is: what should we do about it?  Presumably you've seen many email addresses that are seemingly invalid based on the RFC alone yet work just fine in the wild, and that seems to be because email vendors can and do support addresses that are not conformant.  If we don't use the RFC as-is or feel it's overly restrictive, we could consider relaxing our rules in some way (allowing non-US ASCII characters, for example).